files/blocks/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "blocks",
+  "repo": "astoilkov/jsblocks",
+  "files": {
+    "basePath": "dist/"
+  }
```

Comment:
add also an include parameter for:
"*.min.*", "./mvc/*.min.*", "./query/*.min.*"